src/odemis/driver/xt_client.py

```diff
@@ -1275,13 +1275,24 @@ def start_autostig(self, n_images=17, sweep_range=0.015, dwell_time=3e-6, resolu
 
         Returns
```

Comment:
The docstrings don't match the current format https://github.com/delmic/odemis/blob/c837e587c6356559fc6f4ace9a6d764cdfd0cd05/doc/develop/coding-guidelines.rst

Reply:
This whole file contains a mix of different docstring formats, I'll open a new PR to make sure everything matches the current format.